### DISCUSSION TITLE
getting TimeoutException on DESCRIBE_CONFIGS

### DISCUSSION THREAD

**Discussion by shaharfreidenberg** [provectus/kafka-ui#3444] [Q&A]
I've deployed the Kafka UI helm chart on an EKS cluster.
I'm using IAM authentication to connect to two different MSK clusters. One is in the same VPC as the EKS cluster, and the other one is in a peered VPC and has a security group that allows access from the EKS cluster VPC.

The Kafka UI is able to connect to the MSK cluster that is in the same VPC, but throws an error when trying to connect to the MSK cluster that is in the peered VPC.

<img width="1430" alt="Screen Shot 2023-03-04 at 19 51 59" src="https://user-images.githubusercontent.com/29702108/222921182-a873259f-03a1-4ac6-8987-d39ea79c8a34.png">

Both clusters are configured exactly the same in AWS.

I've tested network connectivity and have also used a Kafka client with the same IAM role to test IAM permissions. 
I was able to connect to both clusters using a Kafka client that is installed in the same EKS cluster as the Kafka UI.

**Comment by Haarolean** [comment]
Hey,

sorry for the issue you encountered.

This'd be hard to judge what's wrong considering a complicated environment.
The least I can say is that we had multiple successful cases of using MSK/serverless, so the issue is related to AWS configuration.
I can also suggest asking around on discord (the link is available in the top of readme file), some might've had something similar.
